src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -427,20 +426,27 @@ protected void Stop()
 
                     return;
                 }
+            }
+            else if (customPipeNameIsSet)
+            {
+                if (runspaceVersion.Version.Major < 6 && runspaceVersion.Version.Minor < 2)
```

Comment:
Although this works, it strikes me as less obvious than `runspaceVersion.Version <= new Version(6, 1)`, which in the context of remoting is worth the minor allocation overhead to improve readability

Reply:
(It's also something that could be cached statically if we really wanted to)

Reply:
fixed.